qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -101,3 +101,18 @@
             driver_name = pvpanic
             device_name = "QEMU PVPanic Device"
             device_hwid = '"ACPI\QEMU0001"'
+        - with_vioinput:
+            no Host_RHEL.m6 Host_RHEL.m7.u0 Host_RHEL.m7.u1 Host_RHEL.m7.u2 Host_RHEL.m7.u3
+            no Win2008..sp2
+            driver_name = vioinput
+            device_name = "VirtIO Input Driver"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1052"'
+            inputs = input1
```

Comment:
Hi, as the patch https://github.com/avocado-framework/avocado-vt/pull/1737 Example:
  <qemu command line>
  -device virtio-mouse-pci,id=input_mouse1,bus=pci.0,addr=0x6 \
  -device virtio-keyboard-pci,id=input_keyboard1,bus=pci.0,addr=0x7
The virtio-mouse/keyboard/table device id is input_mouse1/input_keyboard1/input_tablet1, but it's all show as input_input1 in my tested result.
could you check if some thing wrong at here?

the tested qemu-command line as: -device virtio-mouse-pci,id=input_input1,bus=pcie_root_port_2,addr=0x0

the tested log location as: http://10.66.4.244/kvm_autotest_job_log/?jobid=3057126

Reply:
It is the expected result, because the device is assigned with the id 'input1' here.